WordPressUI/Extensions/UIImageView+Gravatar.swift

```diff
@@ -110,15 +153,29 @@ extension UIImageView {
     /// P.s.:
     /// Hope buddah, and the code reviewer, can forgive me for this hack.
     ///
-    @objc
-    public func overrideGravatarImageCache(_ image: UIImage, rating: GravatarRatings, email: String) {
-        guard let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating.stringValue()) else {
+    @available(*, deprecated)
+    @objc public func overrideGravatarImageCache(_ image: UIImage, rating: GravatarRatings, email: String) {
+        guard let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating) else {
             return
         }
 
+        listenForGravatarChanges()
         overrideImageCache(for: gravatarURL, with: image)
     }
 
+    /// Updates the gravatar image for the given email, and notifies all gravatar image views
+    ///
+    /// - Parameters:
+    ///   - image: the new UIImage
+    ///   - email: associated email of the new gravatar
+    @objc public func updateGravatar(image: UIImage, email: String?) {
+        self.image = image
+        guard let email = email, let gravatarURL = gravatarUrl(for: email, size: Defaults.imageSize, rating: .x) else {
+            return
+        }
+        NotificationCenter.default.post(name: .GravatarImageUpdateNotification, object: self, userInfo: ["email": email, "image": image])
```

Comment:
Same as above, prob. a good idea to turn "email" and "image" into actual constants.

Reply:
Done and done!